view/theme/frio/templates/paginate.tpl

```diff
@@ -0,0 +1,13 @@
+{{if $pager}}
+<div class="{{$pager.class}}">

```

Comment:
It would be good to have an additional common class here (or maybe ID - but I don't know if more than one pager can be on a page)